src/concrete/ml/onnx/convert.py

```diff
@@ -158,15 +163,21 @@ def get_equivalent_numpy_forward_from_torch(
 
 def get_equivalent_numpy_forward_from_onnx(
     onnx_model: onnx.ModelProto,
-    check_model: bool = True,
+    q_x: numpy.ndarray = None,
```

Comment:
q_x is optional though (typing I mean) !

Reply:
ok